graphrag/config/enums.py

```diff
@@ -86,6 +86,7 @@ class TextEmbeddingTarget(str, Enum):
 
     all = "all"
     required = "required"
+    none = "none"
```

Comment:
Is this valid? We would still need entity description_embeddings for local search to work

Reply:
@natoverse wanted to implement this version with no embeddings at all for a specific case

Reply:
We should just update the init file, so it add none to the options then